assembly/spdx-license-knowledge-base/pom.xml

```diff
@@ -61,8 +61,17 @@
                     <groupId>log4j</groupId>
                     <artifactId>log4j</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
These POM changes deserve some extra justification / explanation in the commit message body, i.e. why are they required *now* but not before?